src/App.jsx

```diff
@@ -3,15 +3,30 @@ import Header from './commonComponents/Header/Header';
 import AppWrapper from './commonComponents/AppWrapper/AppWrapper';
 import Search from './commonComponents/Search/Search';
 import './sass/App.sass';
+import {BrowserRouter, Route, Switch} from "react-router-dom";
+import BeerPage from "./commonComponents/BeerPage/BeerPage";
+import FavoritesPage from "./commonComponents/FavoritesPage/FavoritesPage";
 
 function App() {
   return (
+      <BrowserRouter>
     <div className="app">
-      <Header />
-      <Search />
-      <AppWrapper />
+        <Header />
+        <Switch>
+            <Route path="/beer">
+                <BeerPage beerName={"Mocha"}/>
+            </Route>
+            <Route path='/favorites'>
+                <FavoritesPage/>
+            </Route>
+            <Route path="/">
+                <Search />
+                <AppWrapper />
```

Comment:
AppWrapper is not appropriate here. Here should be only HomePage with search inside it

Reply:
Done